src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php

```diff
@@ -15,12 +15,19 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
 /**
- * AddConsoleCommandPass.
+ * Registers console commands.
  *
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ *
+ * @deprecated since version 3.2, to be removed in 4.0. Use Symfony\Component\Console\DependencyInjection\AddConsoleCommandPass instead.
  */
 class AddConsoleCommandPass implements CompilerPassInterface
 {
+    public function __construct()
+    {
+        @trigger_error(sprintf('%s is deprecated as of 3.2 and will be removed in 4.0. Use Symfony\Component\Console\DependencyInjection\AddConsoleCommandPass instead.', __CLASS__), E_USER_DEPRECATED);
```

Comment:
this should be jus below the namespace declaration, see other places in the code base